tests/sentry/web/frontend/test_auth_login.py

```diff
@@ -381,3 +381,120 @@ def test_registration_subscribe_to_newsletter(self):
         assert results[0].list_id == newsletter.get_default_list_id()
         assert results[0].subscribed
         assert not results[0].verified
+
+
+def provision_middleware():
+    # TODO: to be removed once CustomerDomainMiddleware is activated.
+    middleware = list(settings.MIDDLEWARE)
+    if "sentry.middleware.customer_domain.CustomerDomainMiddleware" not in middleware:
+        index = middleware.index("sentry.middleware.auth.AuthenticationMiddleware")
+        middleware.insert(index + 1, "sentry.middleware.customer_domain.CustomerDomainMiddleware")
+    return middleware
+
+
+class AuthLoginCustomerDomainTest(TestCase):
+    @fixture
+    def path(self):
+        return reverse("sentry-login")
+
+    def test_login_valid_credentials(self):
+        # load it once for test cookie
+        self.client.get(self.path)
+
+        resp = self.client.post(
+            self.path,
+            {"username": self.user.username, "password": "admin", "op": "login"},
+            HTTP_HOST="albertos-apples.testserver",
+            follow=True,
+        )
+        assert resp.status_code == 200
+        assert resp.redirect_chain == [
+            (reverse("sentry-login"), 302),
+            ("/organizations/new/", 302),
+        ]
+
+    def test_login_valid_credentials_with_org(self):
+        self.create_organization(name="albertos-apples", owner=self.user)
+        # load it once for test cookie
+        self.client.get(self.path)
+
+        resp = self.client.post(
+            self.path,
+            {"username": self.user.username, "password": "admin", "op": "login"},
+            HTTP_HOST="albertos-apples.testserver",
+            follow=True,
+        )
+        assert resp.status_code == 200
+        assert resp.redirect_chain == [
+            (reverse("sentry-login"), 302),
+            ("/organizations/albertos-apples/issues/", 302),
```

Comment:
If the login was on the customer domain, why do we redirect to an orgslug path? Longer term will this be `/issues`?

Reply:
> If the login was on the customer domain, why do we redirect to an orgslug path?

@markstory The hostname is currently `albertos-apples.testserver` (as set by `HTTP_HOST`) and remains unchanged during the redirects. Only the path changes.

The non-customer domain version of this test is in https://github.com/getsentry/sentry/blob/5c79c39552bc747fdae8ee14b9a6aab9dbdb1bc4/tests/sentry/web/frontend/test_auth_login.py#L72-L86 without `HTTP_HOST` explicitly set.

> Longer term will this be `/issues`?

Yes, that'll be the ideal route to be used long term.